x/upgrade/client/cli/query.go

```diff
@@ -1,98 +1,85 @@
 package cli
 
 import (
-	"encoding/binary"
+	"context"
 	"fmt"
 
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
 
 	"github.com/spf13/cobra"
 
 	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/cosmos/cosmos-sdk/codec"
 )
 
-// GetPlanCmd returns the query upgrade plan command
-func GetPlanCmd(storeName string, cdc *codec.Codec) *cobra.Command {
+// GetCurrentPlanCmd returns the query upgrade plan command
+func GetCurrentPlanCmd(clientCtx client.Context) *cobra.Command {
```

Comment:
note: just a rename to follow same convention everywhere in x/upgrade: `*CurrentPlan*` & `*AppliedPlan*`